lib/inspec/resources/nginx_conf.rb

```diff
@@ -54,6 +54,15 @@ def to_s
       "nginx_conf #{@conf_path}"
     end
 
+    def method_missing(name)
```

Comment:
Please also define `respond_to_missing?` otherwise some weird issues can occur. 
https://github.com/inspec/inspec/issues/4591 was a previous issue that shows the errors that might occur.
https://thoughtbot.com/blog/always-define-respond-to-missing-when-overriding describes what to define and why.

Reply:
I just added them in d31e8e2f0. Note that almost none of the other resources implementing `method_missing` implement `respond_to_missing?`, and from what I noticed from tests they should be cautious and also exclude `to_` methods, and possibly others.